File: UINavigationExtensionDemo/Feature/Advanced/Controllers/ViewController01_DisablePopGesture.m
//
//  ViewController01_DisablePopGesture.m
//  UINavigationExtensionDemo
//
//  Created by lidan on 2020/10/26.
//

#import <UINavigationExtension/UINavigationExtension.h>

#import "ViewController01_DisablePopGesture.h"

@interface ViewController01_DisablePopGesture ()

@end

@implementation ViewController01_DisablePopGesture

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (BOOL)ue_disableInteractivePopGesture {
    return YES;
}

@end

File: Example/UNXNavigatorSwiftDemo/Advanced/ViewController01_DisablePopGesture.swift
//
//  ViewController01_DisablePopGesture.swift
//  UNXNavigatorSwiftDemo
//
//  Created by Leo Lee on 2020/11/23.
//

import UIKit

class ViewController01_DisablePopGesture: BaseTableViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }
    
    override var unx_disableInteractivePopGesture: Bool {
        return true
    }
    
}
